# Remove default flag CC and use SPKSRC_CFLAGS and SPKSRC_LDFLAGS
--- Makefile.orig	2014-09-01 09:59:12.076052667 +0200
+++ Makefile	2014-09-01 10:04:52.303775427 +0200
@@ -107,7 +107,7 @@

 #### Toolchain options.
 # GCC is normally used both for compiling and linking.
-CC = gcc
+#CC = gcc
 LD = $(CC)

 #### Debug flags (typically "-g").
@@ -190,13 +190,13 @@
 # These CFLAGS contain general optimization options, CPU-specific optimizations
 # and debug flags. They may be overridden by some distributions which prefer to
 # set all of them at once instead of playing with the CPU and DEBUG variables.
-CFLAGS = $(ARCH_FLAGS) $(CPU_CFLAGS) $(DEBUG_CFLAGS) $(SPEC_CFLAGS)
+CFLAGS = $(ARCH_FLAGS) $(CPU_CFLAGS) $(DEBUG_CFLAGS) $(SPEC_CFLAGS) $(SPKSRC_CFLAGS)

 #### Common LDFLAGS
 # These LDFLAGS are used as the first "ld" options, regardless of any library
 # path or any other option. They may be changed to add any linker-specific
 # option at the beginning of the ld command line.
-LDFLAGS = $(ARCH_FLAGS) -g
+LDFLAGS = $(ARCH_FLAGS) -g $(SPKSRC_LDFLAGS)

 #### Target system options
 # Depending on the target platform, some options are set, as well as some
